lib/app/bloc/app_bloc.dart

```diff
@@ -30,10 +29,17 @@ class AppBloc extends Bloc<AppEvent, AppState> {
   Stream<AppState> mapEventToState(AppEvent event) async* {
     if (event is AppUserChanged) {
       yield _mapUserChangedToState(event, state);
-    } else if (event is AppLogoutRequested) {
+      return;
+    }
+
+    if (event is AppLogoutRequested) {
       _authenticationRepository.logOut();
-    } else if (event is AppUserDeleteRequested) {
+      return;
+    }
```

Comment:
daloby sie zrobic? :

if (event is AppLogoutRequested) {
      return;
}

if (event is AppUserDeleteRequested) {
      return;
    }

if (event is AppUserChanged) {
      yield _mapUserChangedToState(event, state);
    }

albo cos takiego zeby calkiem esle if sie pozbyc?

if (event is AppLogoutRequested) {
      _authenticationRepository.logOut();
    }

Reply:
za malo kodu zaznaczylem ale wiesz o ktory fragment chodzi

Reply:
A jakbym zrobił tak jak napisałeś, to jak będzie event AppLogoutRequested to najpierw wywoła się if z returnem i wyjdzie z funkcji, a nie wywoła się w ogóle drugi if dla tego eventu i nie wyloguje użytkownika.

Reply:
dobra

Reply:
to git